.travis.yml

```diff
@@ -64,10 +64,13 @@ jobs:
   include:
     - stage: lint
       sudo: false
-      install: pip install -r requirements-lint.txt
+      install:
+        - pip install -r requirements-lint.txt
+        - pip install -r requirements-mypy.txt
```

Comment:
perhaps simply merge linter and mypy requirements into the linter one

Reply:
That's done and this diff disappears after all.